src/components/top_bar/top_bar.ts

```diff
@@ -164,31 +152,59 @@ export class TopBar extends Component<Props, SpreadsheetChildEnv> {
     PaintFormatButton,
     BorderEditorWidget,
     TableDropdownButton,
+    Popover,
```

Comment:
Remove the unused one (ColorPickerWidget, ColorPicker, FontSizeEditor, ActionButton, PaintFormatButton, BorderEditorWidget and TableDropdownButton I think)